test/indexing.jl

```diff
@@ -74,8 +74,8 @@
         m = [1.0 2.0; 3.0 4.0]
 
         @test m[(1,2), (1,2)] === @SMatrix [1.0 2.0; 3.0 4.0]
-        @test m[1, (1,2)] === @SVector [1.0, 2.0]
-        @test m[(1,2), 1] === @SVector [1.0, 3.0]
+        @test m[1, (1,2)] ===  (1.0, 2.0)
+        @test m[(1,2), 1] ===  (1.0, 3.0)
```

Comment:
This certainly seems useful and sensible as it stands.  My only question is how the type change here generalizes to more dimensions.  That is, what type should the following return?

```
m[(1,2), (1,2)]
```

Maybe there's no good answer to this / maybe we don't care!  I don't claim to have thought deeply about it.  What do you guys think?